addons/main/XEH_preInit.sqf

```diff
@@ -11,4 +11,30 @@ GVAR(shareHandlers) = [];
 
 GVAR(blockSuppressionModelCache) = false call CBA_fnc_createNamespace;
 
+if (isServer) then {
+    GVAR(versionLoadedOnServer) = QUOTE(VERSION_STR);
+    publicVariable QGVAR(versionLoadedOnServer);
+} else {
+    0 spawn {
+        waitUntil {time > 0};
+        if (!isNil QGVAR(versionLoadedOnServer) && GVAR(versionLoadedOnServer) isEqualTo QUOTE(VERSION_STR)) exitWith {};
```

Comment:
Needs lazy eval, otherwise it's still comparing nil to string

Reply:
I blame Joko

Reply:
fixed on master ^^ thanks. but stupid SQF 🤣

Reply:
> stupid SQF

Amen to that. _Won't be getting much better ..._